### PR TITLE
Make Definition Clearer

### PR DESCRIPTION
**Pull request by lawrencejon**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
Also add the following lines to this.

"In many forms there is a `submit` button which when clicked points out the empty fields left out which need to be filled before submitting.
this is usually achieved by the `required` attribute."

good work. 👍

File: guide/english/html/attributes/required/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Required
 ## Required
 
 
-The HTML `required` attribute is used in an input element to make the input field in a form required to submit the form. This means that the user must fill in the field. <br /> 
+The HTML `required` attribute specifies that an input field must be filled out before a form can be submitted. This means that the user must fill in the field. <br />
```
Comment (thecodingaviator) [inline guide/english/html/attributes/required/index.md]:
```suggestion
The HTML `required` attribute specifies that an input field must be compulsorily filled out before a form can be submitted. <br /> 
```

Comment (RandellDawson) [reply guide/english/html/attributes/required/index.md]: @thecodingaviator While this does shorten the text, `compulsorily` is not a word that everyone uses in their everyday life.  May be something like:
```suggestion
The HTML `required` attribute specifies that filling out an input field is mandatory before a form can be submitted. <br />
```

Comment (thecodingaviator) [reply guide/english/html/attributes/required/index.md]: @RandellDawson it definitely does. Compulsprily is not something many would be familiar with

**Comment by RandellDawson** [comment]
@lawrencejon OK, I think both @thecodingaviator and I agree to make the suggested changes which I made above, unless have an issue with my suggestion.  We are open to discussing it further.

**Review by thecodingaviator — APPROVED** [review]
LGTM! 👍

**Review by RandellDawson — APPROVED** [review]

**Review by Manish-Giri — APPROVED** [review]
LGTM. Thank You for your contribution.

**Pull request merged**